### ISSUE TITLE
`website` branch got force pushed over

### ISSUE THREAD

**Issue by gahooa** [appcove/developer-software#8]
Last night I pushed a single commit to the `website` branch so it had no shared history with master.

Today it has all of master plus 1 commit...

The scripts need updated to 
1. checkout the `website` branch
2. Copy the .deb files into it
3. Push it (not with force)

**Comment by DilecPadovani** [comment]
This it to solve this error: 
``` bash
To https://github.com/appcove/developer-software
 ! [rejected]        website -> website (fetch first)
error: failed to push some refs to 'https://github.com/appcove/developer-software'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

**Comment by DilecPadovani** [comment]
@gahooa  Done it. Now `website` branch is an ophan of main.

**Comment by gahooa** [comment]
Yeah they shoud just not share any history.  Website is a different purpose than master.   It should have a continuous history of it's own.

**Comment by gahooa** [comment]
@DilecPadovani almost there, but still has stray files (`.gitmodules`, and `.github`)

I pushed a commit to `website2` to illustrate the way it should be.  Please take a look at that.

**Comment by gahooa** [comment]
Hi @DilecPadovani 

For some reason, .github and .gitmodules are still on this branch. 
Also, the branch should indeed have a history.

Here is my suggestion:
1. Create a new repo
2. run git remote add origin ...
3. make an empty commit
4. run git push origin HEAD:website

Update the code in .github such that it does NOT create an orphan branchg, it just checks out the website branch, updates the files in the ubuntu folder, and commits it.

Then we have:
1. no stray files in website
2. a history of binaries

**Comment by DilecPadovani** [comment]
In order to have a history of binaries, I suggest trying to use the GitHub release feature

**Comment by gahooa** [comment]
@DilecPadovani I'd rather not switch to a different system at the moment.  This is a simple fix.  See the 2 files below that do not belong:

![Screenshot from 2022-08-18 00-06-56](https://user-images.githubusercontent.com/144110/185291615-52a84079-849f-4ff8-96a2-4eec000d286b.png)

Please implement a fix here such that you reset the branch a single time manually, and then stop using the orphan option, and all will be fine.

**Comment by DilecPadovani** [comment]
> Hi @DilecPadovani
> 
> For some reason, .github and .gitmodules are still on this branch. Also, the branch should indeed have a history.
> 
> Here is my suggestion:
> 
> 1. Create a new repo
> 2. run git remote add origin ...
> 3. make an empty commit
> 4. run git push origin HEAD:website
> 
> Update the code in .github such that it does NOT create an orphan branchg, it just checks out the website branch, updates the files in the ubuntu folder, and commits it.
> 
> Then we have:
> 
> 1. no stray files in website
> 2. a history of binaries

I don't really like the idea of having a separate repo just for the website to be accessed.

**Comment by gahooa** [comment]
@DilecPadovani reminder to finish this
